src/pykeen/models/unimodal/conv_kb.py

```diff
@@ -81,66 +78,23 @@ def __init__(
         preferred_device: DeviceHint = None,
         num_filters: int = 400,
         random_seed: Optional[int] = None,
-        regularizer: Optional[Regularizer] = None,
     ) -> None:
         """Initialize the model.
 
         To be consistent with the paper, pass entity and relation embeddings pre-trained from TransE.
         """
         super().__init__(
             triples_factory=triples_factory,
+            interaction=ConvKBInteraction(
+                hidden_dropout_rate=hidden_dropout_rate,
+                embedding_dim=embedding_dim,
+                num_filters=num_filters,
+            ),
             embedding_dim=embedding_dim,
             loss=loss,
             automatic_memory_optimization=automatic_memory_optimization,
             preferred_device=preferred_device,
             random_seed=random_seed,
-            regularizer=regularizer,
         )
-
-        self.num_filters = num_filters
-
-        # The interaction model
-        self.conv = nn.Conv2d(in_channels=1, out_channels=num_filters, kernel_size=(1, 3), bias=True)
-        self.relu = nn.ReLU()
-        self.hidden_dropout = nn.Dropout(p=hidden_dropout_rate)
-        self.linear = nn.Linear(embedding_dim * num_filters, 1, bias=True)
-
-    def _reset_parameters_(self):  # noqa: D102
-        # embeddings
+        self.regularizer = self._instantiate_default_regularizer(parameters=list(self.interaction.parameters()))
```

Comment:
Why isn't this stored in the weight_regularizers list?

Reply:
ccfeb08